content/courses/native-onchain-development/serialize-instruction-data-frontend.md

```diff
@@ -255,11 +255,18 @@ const instruction = new web3.TransactionInstruction({
 
 transaction.add(instruction);
 
-web3.sendAndConfirmTransaction(connection, transaction, [player]).then(txid => {
+try {
+  const transactionId = await web3.sendAndConfirmTransaction(
+    connection,
+    transaction,
+    [player],
+  );
   console.log(
-    `Transaction submitted: https://explorer.solana.com/tx/${txid}?cluster=devnet`,
+    `Transaction submitted: https://explorer.solana.com/tx/${transactionId}?cluster=devnet`,
```

Comment:
Check out https://github.com/solana-developers/helpers?tab=readme-ov-file#get-a-solana-explorer-link-for-a-transaction-address-or-block

Reply:
We should also move to just importing what we need from web3 (it's more efficient due to tree shaking and stops us needing to type `web3.` all the time).

Reply:
> We should also move to just importing what we need from web3 (it's more efficient due to tree shaking and stops us needing to type `web3.` all the time).

yes i have made this change in all the PR's - #346, #347 , #348

Reply:
> Check out https://github.com/solana-developers/helpers?tab=readme-ov-file#get-a-solana-explorer-link-for-a-transaction-address-or-block

implemented ✅